iceoryx_dds/test/moduletests/test_dds_to_iox.cpp

```diff
@@ -47,6 +47,14 @@ TEST_F(DDS2IceoryxGatewayTest, ChannelsAreCreatedForConfiguredServices)
     iox::config::GatewayConfig config{};
     config.m_configuredServices.push_back(iox::config::GatewayConfig::ServiceEntry{testService});
 
+    auto mockPublisher = createMockIceoryxTerminal(testService, iox::popo::PublisherOptions());
+    EXPECT_CALL(*mockPublisher, offer).Times(1);
+    stageMockIceoryxTerminal(std::move(mockPublisher));
+
+    auto mockDataReader = createMockDDSTerminal(testService);
+    EXPECT_CALL(*mockDataReader, connect).Times(1);
+    stageMockDDSTerminal(std::move(mockDataReader));
```

Comment:
@ithier This is weird. When the unexpected mock calls are fixed, all three test in this file are equal. I guess during a refactoring of iceoryx the behavior changed and these other tests don't make sense anymore. I would like to remove them if you don't think they are important

Reply:
DDS is at the moment not safety critical and from this point of view I share your opinion and would just remove the tests.